data/restli-csharp/datagenerator/src/test/GeneratedDataTemplate/com/linkedin/restli/datagenerator/integration/PrimitiveOptionalRecord.cs

```diff
@@ -0,0 +1,181 @@
+using System.Collections.Generic;
+using System;
+
+// DO NOT EDIT - THIS FILE IS GENERATED BY restli-csharp
+// Generated from com/linkedin/restli/datagenerator/integration/PrimitiveOptionalRecord.pdsc
+
+namespace com.linkedin.restli.datagenerator.integration
+{
+  /// <summary>
+  /// Contains fields of all supported primitive schema types
+  /// Every field is optional
+  /// </summary>
+  public class PrimitiveOptionalRecord
+  {
+    /// <summary>I will make the doc look casual</summary>
+    // optional, has default value
```

Comment:
This comment is not from PrimitiveOptionalRecord.pdsc? We should only generate comment defined in pdsc `docs`. Reference to my previous comment style.